index.js

```diff
@@ -1,17 +1,65 @@
 class SortedList {
-  constructor() {}
+   
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort( (a,b) => {
+      return a-b;
+    });
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < 0 || pos > this.items.length) {
+      throw new Error ('OutOfBounds');
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0) {
+      throw new Error('EmtySortedList');
+    } else {
+      return Math.max(...this.items);
```

Comment:
here you can use two aproachs. A new one which is a Math.max() inside a reduce method (take a look at it) or simply sort the array and return the last or fist number depending of how you sort it.

Hope it made sense.